light-client/src/node.rs

```diff
@@ -31,6 +31,19 @@ impl Node {
         })
     }
 
+    pub fn with_portal(config: Arc<Config>, portal_rpc: R) -> Result<Self, NodeError> {
+        let checkpoint_hash = &config.checkpoint.as_ref().unwrap();
+
+        let consensus =
+            ConsensusLightClient::with_custom_rpc(portal_rpc, checkpoint_hash, config.clone());
+
+        Ok(Node {
+            consensus,
+            config,
+            history_size: 64,
```

Comment:
define as const?

Reply:
Good catch, this is a leftover,  removed.